modules/tribes/client/components/JoinButton.js

```diff
@@ -59,7 +59,9 @@ export default function JoinButton({ tribe, user, onUpdated, ...rest }) {
 
   const [isUpdating, setIsUpdating] = useState(false);
 
-  const isMemberInitial = user && user.memberIds && user.memberIds.indexOf(tribe._id) > -1;
+  // This should always be true or false, even when user is null.
+  // Therefore we convert the output to boolean.
+  const isMemberInitial = Boolean(user && user.memberIds && user.memberIds.indexOf(tribe._id) > -1);
```

Comment:
Nitpick; 
```
user.memberIds.indexOf(tribe._id) > -1
```
can also written with little more modern [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes): 
```
! user.memberIds.includes(tribe._id)
```
...up to you. :-)

Reply:
What do you think about `(_.)get(user, ['memberIds'], []).includes(tribe._id)`? Are we ok with `import get from lodash/get` and such?

Reply:
Oh, actually we could enable optional chaining JS proposal to make this simpler: https://github.com/Trustroots/trustroots/pull/1158

```js
(user?.memberIds || []).includes(tribe._id))
```

What do you think?

Not opposed to including Lodash in frontend if it makes code more readable but nice if we can avoid it like this.

Reply:
I like this very much!

I just noticed I had used `import get from lodash/get` in the frontend before, so it will be great to replace that with this new syntax, too! :tada:

Reply:
fyi, for frontend lodash imports `lodash.get` means it doesn't have to load the whole of lodash.

Reply:
improved with the optional chaining and `array.includes` in 8f13a627b6f84a5f8af89177045725.

I'm happy to have learned the new syntax of #1158! :tada: